src/java/us/kbase/mobu/ModuleBuilder.java

```diff
@@ -437,6 +445,18 @@ private static int runRunCommand(RunCommandArgs runArgs, JCommander jc) {
         }
     }
 
+    private static int runShellCommand(ShellCommandArgs shellArgs, JCommander jc) {
+        try {
+            ShellLauncher launcher = new ShellLauncher();
+            launcher.exec();
+            return 0;
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Add a verbose argument like the other commands, and only print the trace if true

Reply:
Alternatively just pass the exception and verbose boolean to showError() and put all the error display logic in one spot